tensorbay/cli/utility.py

```diff
@@ -19,8 +20,13 @@
 from ..client import config as client_config
 from ..client.dataset import DatasetClient, FusionDatasetClient
 from ..client.gas import DatasetClientType
+from ..client.log import dump_request_and_response
+from ..client.requests import logger
+from ..exception import InternalServerError, TensorBayException
 from .tbrn import TBRN
 
+_Callable = TypeVar("_Callable", bound=Callable[..., Any])
```

Comment:
all `_implement` functions return None, will it be better to change the return type from `Any` to `None`?